tools/ci/tc/tasks/test.yml

```diff
@@ -128,6 +128,10 @@ components:
         - affected_tests
 
 tasks:
+  # The scheduling order of tasks is NOT determined by the order in which they
+  # are defined, but by their dependencies (depends-on); sink-task is always
+  # scheduled the last.
```

Comment:
Nit; mentioning sink-task here is more confusing than not imo; it's not always scheduled (only for PRs).

If we want to document sink-task, let's write a comment above its definition noting that its dependencies are added programmatically?